osf/migrations/0136_preprint_node_divorce.py

```diff
@@ -89,317 +89,11 @@ def reverse_func(apps, schema_editor):
-            SELECT C.user_id as osfuser_id, G.id as group_id
-            FROM osf_preprint as P, osf_abstractnode as N, osf_contributor as C, auth_group as G
-            WHERE P.node_id IS NOT NULL
-            AND P.node_id = N.id
-            AND C.node_id = N.id
-            AND C.read = TRUE
-            AND C.write = TRUE
-            AND C.admin = TRUE
-            AND G.name = 'preprint_' || P.id || '_admin';
-
-            -- Add all the tags on nodes to their corresponding preprint
-            INSERT INTO osf_preprint_tags (preprint_id, tag_id)
-            SELECT P.id, T.tag_id
-            FROM osf_preprint AS P, osf_abstractnode AS N, osf_abstractnode_tags as T
-            WHERE P.node_id IS NOT NULL
-            AND P.node_id = N.id
-            AND T.abstractnode_id = N.id;
-
-            -- Update preprint region to be the same as the node's region
-            UPDATE osf_preprint
-            SET region_id = NS.region_id
-            FROM addons_osfstorage_nodesettings NS
-            WHERE osf_preprint.node_id = NS.owner_id;
-
-            -- Create a root folder for each preprint
-            INSERT INTO osf_basefilenode
-            (created, modified, _id, type, target_content_type_id, target_object_id, provider, name,
-              _path, _materialized_path, is_root, _history)
-            SELECT current_timestamp,
-                current_timestamp,
-                generate_object_id(),
-                'osf.osfstoragefolder',
-                CT.id,
-                P.id,
-               'osfstorage',
-               '',
-               '',
-               '',
-               true,
-               '[]'
-            FROM osf_preprint as P, django_content_type as CT
-            WHERE P.node_id IS NOT NULL
-            AND CT.model = 'preprint' and CT.app_label = 'osf';
-
-            -- Move the node's preprint file target from the node -> preprint, and
-            -- set the file's parent as the preprint's root_folder
-            UPDATE osf_basefilenode Fi
-            SET target_object_id = P.id,
-              target_content_type_id = CT.id,
-              parent_id = Fo.id
-            FROM osf_preprint P, osf_abstractnode N, django_content_type CT, osf_basefilenode Fo
-            WHERE P.node_id = N.id
-            and P.node_id IS NOT NULL
-            and N.preprint_file_id = Fi.id
-            and CT.model = 'preprint' and CT.app_label = 'osf'
-            and Fo.is_root = TRUE
-            and Fo.target_object_id = P.id
-            and Fo.target_content_type_id = CT.id;
-
-            -- Set the preprint primary file as the node's preprint file
-            UPDATE osf_preprint P
-            SET primary_file_id = N.preprint_file_id
-            FROM osf_abstractnode N
-            WHERE P.node_id = N.id
-            AND P.node_id IS NOT NULL;
-
-            -- Update primary_file entries in the PageCounter table to reference the preprint _id
```

Comment:
This is the only removed block, everything else was just separated into individual statements and put in the `RunSQL`. Makes it ~~easier~~ possible to track what's running, and easier to kill if necessary.